src/CreateBird/CreateBird.jsx

```diff
@@ -1,25 +1,33 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
+import { postBird } from '../services/birds.js';
 
-export default class CreateBird extends Component {
+class CreateBird extends Component {
     state = {
         bird: '',
-        season: '',
-        spotted: ''
+        season: ''
     }
 
     handleChange = (e) => {
```

Comment:
destructure target off of e here.

Also, if you want to use event here, either call tihs one event or the handleSubmit one e. Keep it consistent.